test_vm/vagrant_config_defaults.rb

```diff
@@ -2,6 +2,6 @@
   {nfs: true, nfs_version: 4, mount_options: ["noatime", "nodiratime"]}
 
 # Default options
-CONFIG['memory'] = "1100"
+CONFIG['memory'] = "2048"
```

Comment:
Perhaps I'm going mad. I've just run it through fine with `1100`. I'll revert to `1100`.